tests/utils/test_manifest.py

```diff
@@ -66,7 +66,7 @@ def test_read_manifest_entry(generated_manifest_entry_file: str) -> None:
 
     assert manifest_entry.status == ManifestEntryStatus.ADDED
```

Comment:
nit: add a test to check the string representation fo manifest_entry to show that the new field is called `sequence_number`

Reply:
I think this is already getting validated in [here](https://github.com/apache/iceberg-python/pull/900/files#diff-9a5a57e7ec723cc4ab6056ce1921f291a501158322d2e6ebf817d568c9621f4bL187).
At the end of the test we are comparing the contents of avro file with the expected `ManifestEntry` object.
Please correct me if my understanding is wrong.

Reply:
ah i see, thank you. it is implicitly checked in that test